### PR TITLE
a few details in master reference file

### PR DESCRIPTION
**Pull request by fchapoton**
in particular, using the arxiv role

### :memo: Checklist

- [x] The title is concise and informative.
- [x] The description explains in detail what this PR is about.

### CODE REVIEW

File: src/doc/en/reference/references/index.rst [hunk 1]
```diff
@@ -1115,18 +1116,18 @@ REFERENCES:
...
+             equations: I. Fibonacci and Lucas perfect powers.* Annals
              of Math, 2006.
 
 .. [BMSS2006] Alin Bostan, Bruno Salvy, François Morain, Éric Schost.
-              Fast algorithms for computing isogenies between elliptic
-              curves. [Research Report] 2006, pp.28. <inria-00091441>
-              https://arxiv.org/pdf/cs/0609020.pdf
+              *Fast algorithms for computing isogenies between elliptic
+              curves*. [Research Report] 2006, pp.28. <inria-00091441>
+              `arxiv:`cs/0609020`
```
Comment (dcoudert) [inline src/doc/en/reference/references/index.rst]: should be `:arxiv:`

File: src/doc/en/reference/references/index.rst [hunk 2]
```diff
@@ -1595,12 +1596,13 @@ REFERENCES:
...
 
-.. [CFZ2000] \J. Cassaigne, S. Ferenczi, L.Q. Zamboni, Imbalances in
-             Arnoux-Rauzy sequences, Ann. Inst. Fourier (Grenoble)
+.. [CFZ2000] \J. Cassaigne, S. Ferenczi, L.Q. Zamboni, *Imbalances in
+             Arnoux-Rauzy sequences*, Ann. Inst. Fourier (Grenoble)
              50 (2000) 1265--1276.
 
-.. [CFZ2002] Chapoton, Fomin, Zelevinsky - Polytopal realizations of
-             generalized associahedra, :arxiv:`math/0202004`.
+.. [CFZ2002] Chapoton, Fomin, Zelevinsky - *Polytopal realizations of
```
Comment (dcoudert) [inline src/doc/en/reference/references/index.rst]: add first names ?

**Review by dcoudert — CHANGES_REQUESTED** [review]

**Review by dcoudert — APPROVED** [review]
LGTM.

**Comment by fchapoton** [comment]
merci David ! :christmas_tree:

**Pull request merged**